### DISCUSSION TITLE
Bindable is ambiguous for type lookup

### DISCUSSION THREAD

**Discussion by ba01ei** [pointfreeco/swift-composable-architecture#3358] [General]
I get this error with Xcode 16 beta and the TCA 1.14.0, because both SwiftUI & TCA defined `@Bindable`, so I have to change it to `@ComposableArchitecture.Bindable var store` instead, which works but doesn't look as nice. Is there a plan to rename TCA Bindable, or just remove TCA Bindable and rely on SwiftUI Bindable (once Xcode 16 is out of beta)?

**Comment by stephencelis** [comment]
@ba01ei It's actually `@Perception.Bindable`, though because TCA imports Perception you can namespace it either way. We don't have plans on removing this namespace since it's required for pre-iOS 17 projects, and in iOS 17+ projects you should be able to simply use `@Bindable` again.
